code/modules/reagents/Chemistry-Reagents/Blob-Reagents.dm

```diff
@@ -187,7 +173,7 @@ datum/reagent/blob/spacedrugs/reaction_mob(var/mob/living/M as mob, var/method=T
 
 /proc/reagent_vortex(var/mob/living/M as mob, var/setting_type)
 	var/turf/pull = get_turf(M)
-	for(var/atom/movable/X in orange(4,pull))
+	for(var/atom/movable/X in range(4,pull))
```

Comment:
this is intentional super

orange() = everything but the tile it was on, making sorium blob only throw everything but the guy you hit

range() = everything in that radius